appveyor.yml

```diff
@@ -2,7 +2,7 @@
 
 image: Visual Studio 2017
 
-# cache version - netcoreapp.2.0.0-preview3-25426-01
+# cache version - netcoreapp.2.0.0
 cache:
   - '%LocalAppData%\Microsoft\dotnet -> appveyor.yml'
```

Comment:
Why don't we update the caches to depend on `PowerShell.Common.props`.  Then we only have one place to update the version.